code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -221,23 +221,31 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 
 
-/datum/clockwork_scripture/create_object/judicial_visor //Judicial Visor: Creates a judicial visor.
-	descname = "Delayed Area Stun Glasses"
-	name = "Judicial Visor"
-	desc = "Forms a visor that, when worn, will grant the ability to form a flame in your hand that can be activated at an area to smite it, stunning, muting, and damaging the nonfaithful. \
-	Cultists of Nar-Sie will be set on fire, though they will be stunned for half the time."
-	invocations = list("Grant me the flames of Engine!")
-	channel_time = 10
+/datum/clockwork_scripture/stunning_flare //Stunning Flare: Stuns and blinds the target for 10 seconds, and disorients them for 15.
+	descname = "Melee Stun/Blind"
+	name = "Stunning Flare"
+	desc = "Fills your hand with holy light. You can strike a target with it to stun, blind, and disorient them."
+	invocations = list("Bask in his glory!")
+	whispered = TRUE
+	channel_time = 0
 	required_components = list("belligerent_eye" = 2)
 	consumed_components = list("belligerent_eye" = 1)
-	whispered = TRUE
-	object_path = /obj/item/clothing/glasses/judicial_visor
-	creator_message = "<span class='brass'>You form a judicial visor, which is capable of smiting the unworthy.</span>"
-	usage_tip = "The visor has a thirty-second cooldown once used, and the marker it creates has a delay of 3 seconds before exploding."
+	usage_tip = "The flare breaches eye protection but <i>does not affect silicons</i>."
 	tier = SCRIPTURE_DRIVER
-	space_allowed = TRUE
 	sort_priority = 2
 
+/datum/clockwork_scripture/stunning_flare/check_special_requirements()
+	if(!invoker.get_empty_held_index_for_side("l") && !invoker.get_empty_held_index_for_side("r"))
+		invoker << "<span class='warning'>You need a free hand to hold the flare!</span>"
+		return 0
+	return 1
+
+/datum/clockwork_scripture/stunning_flare/scripture_effects()
+	invoker <<  "<span class='brass'>You copy a piece of replicant alloy and command it into a new slab.</span>" //No visible message, for stealth purposes
+	var/obj/item/weapon/ratvars_light/L = new(get_turf(invoker))
```

Comment:
I'd kind of prefer if this used something similar to what the judicial visor uses, with a ranged ability and a cancel button, and delaying the scripture returning until they either use the flare, cancel, or become unable to use it